### PR TITLE
Viz on leap

### LINKED ISSUE #40: Adam - movies ICCS cm2.6 notebook - GitHub

### PR DESCRIPTION
**Pull request by suryadheeshjith**
This PR is based on the draft PR #42. This PR adds a notebook to visualize UNet predictions. This notebook uses data that was previously not ingested in Leap-pangeo. For ease of uploading data in the future, I have added information to do so in the README. 

 - [x] Closes #40 
 - [x] New notebooks are added to `_toc.yml` and show up properly in website preview.

### CODE REVIEW

**Comment by suryadheeshjith** [comment]
@IamShubhamGupto @jbusecke This is the new PR. I hope we can merge these changes now.

File: .gitignore [hunk 1]
```diff
@@ -10,7 +10,12 @@ _build/
...
-notebooks/Source.gv
-notebooks/*.pkl
-notebooks/*.csv*
+book/notebooks/Source.gv
+book/notebooks/*.pkl
+book/notebooks/*.csv*
 dataset*
+src/notebooks/*.nc
+src/notebooks/*.npy
+src/notebooks/*.gif
```
Comment (jbusecke) [inline .gitignore:19]: Can we generally exclude *.gif and the other extensions here? that might help others in the future to not commit anything uneccessary

File: .gitignore [hunk 2]
```diff
@@ -10,7 +10,12 @@ _build/
...
+token.txt
```
Comment (jbusecke) [inline .gitignore:20]: 😱 Good call!

File: README.md [hunk 3]
```diff
@@ -128,4 +128,18 @@ $ git merge upstream/main
 Ideally you will not have any merge conflicts.
 You are now ready to make a new feature branch.
 
+### Using Large Datasets
```
Comment (jbusecke) [inline README.md:131]: Thanks for adding this. This is great.

File: book/assets/Unet_v.gif
Comment (jbusecke) [inline book/assets/Unet_v.gif:1]: I understand that these gifs now need to be commited, but please make sure to add a note on the PR we just discussed (remove the notebooks that need to read from the hub from the ignore), to also remove these again.

**Review by jbusecke — CHANGES_REQUESTED** [review]
This looks great. I had a few nitpicks on the .gitignore, but I consider these optional.

File: .gitignore (hunk 1 above)
Comment (suryadheeshjith) [reply .gitignore:19]: We cannot generally exclude *.gif because some of them are being used in the notebooks.

File: book/assets/Unet_v.gif
Comment (suryadheeshjith) [reply book/assets/Unet_v.gif:1]: Alright

**Review by LaureZanna — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (jbusecke) [reply .gitignore:19]: Ok thanks for clarifing that.

**Review by jbusecke — APPROVED** [review]

**Pull request merged**